httpcore/_async/connection.py

```diff
@@ -52,9 +52,9 @@ def __init__(
         self._keepalive_expiry = keepalive_expiry
         self._http1 = http1
         self._http2 = http2
-        self._retries = retries
-        self._local_address = local_address
         self._uds = uds
+        self._local_address = local_address
+        self._retries = retries
```

Comment:
Same here. Let's avoid distorting the history unneccessarily.

Reply:
the `retries` and `local_address` were not consistent across different classes, therefore this PR tried to keep it the same.
should keep `retries` before `local_address` and `uds` or keep the current?

Reply:
Oh, okay.

The most sensible ordering would be `retries`, `local_address`, `uds`.